### PR TITLE
chore(docs): normalize indendation usage in docs

### PR DESCRIPTION
**Pull request by hypeJunction**
Replaces all triple and quadruple spaces with tabs

### CODE REVIEW

**Comment by jeabakker** [comment]
hmmm, i don't think this is going to work. see https://documentation-style-guide-sphinx.readthedocs.io/en/latest/style-guide.html#whitespaces

**Comment by jeabakker** [comment]
you know you can build the docs locally to check if it will build http://learn.elgg.org/en/stable/contribute/docs.html

**Comment by jeabakker** [comment]
never mind the docs build succesfully

**Comment by hypeJunction** [comment]
Yes, I built everything locally first.

File: docs/contribute/code.rst [hunk 1]
```diff
@@ -60,15 +60,15 @@ Choosing a branch to submit to
...
-Major feature                  master
-Has any breaking change        master
+Security fix					Don't! Email security@elgg.org for guidance.
+Bug fix						1.12 (or 2.1 if the 1.12 fix is too complex)
+Performance					2.x
+Deprecation					2.x
+Minor feature				  2.x
+Major feature				  master
+Has any breaking change		master
 ============================== ============================================
```
Comment (jeabakker) [inline docs/contribute/code.rst]: maybe check this section

File: docs/appendix/upgrade-notes/1.8-to-1.9.rst [hunk 2]
```diff
@@ -119,90 +119,90 @@ Example of the old way:
...
+		elgg_register_plugin_hook_handler('prepare', 'notification:publish:object:photo', 'photos_prepare_notification');
+	}
+
+	/**
+	 * Prepare a notification message about a new photo
+	 *
+	 * @param string						  $hook		 Hook name
+	 * @param string						  $type		 Hook type
+	 * @param Elgg_Notifications_Notification $notification The notification to prepare
+	 * @param array							$params		Hook parameters
```
Comment (jeabakker) [inline docs/appendix/upgrade-notes/1.8-to-1.9.rst]: does every this still align in doc, in the result it doesn't

File: docs/design/database.rst [hunk 3]
```diff
@@ -298,44 +298,44 @@ entity, which is defined as:
...
-        $owner_id = 0,   // The annotation owner, defaults to current user
-        $vartype = ""    // 'text' or 'integer'
-    )
+	public function annotate(
+		$name,			// The name of the annotation type (eg 'comment')
+		$value,		  // The value of the annotation
+		$access_id = 0,  // The access level of the annotation
+		$owner_id = 0,	// The annotation owner, defaults to current user
+		$vartype = ""	// 'text' or 'integer'
+	)
```
Comment (jeabakker) [inline docs/design/database.rst]: also miss alignment

File: docs/design/database.rst [hunk 4]
```diff
@@ -519,21 +519,21 @@ artist has fans, a user is a member of an organization, etc.
...
 
-================  ===========     =========================================
-API name          Models          Represents
-================  ===========     =========================================
-``guid_one``      The subject     Which entity is being bound
-``relationship``  The noun        The type of relationship
-``guid_two``      The target      The entity to which the subject is bound
-================  ===========     =========================================
+================    ===========	    =========================================
+API name            Models		    Represents
```
Comment (jeabakker) [inline docs/design/database.rst]: how does the align

File: docs/design/events.rst [hunk 5]
```diff
@@ -232,28 +232,28 @@ Hook handlers are callables with one of the following prototypes:
...
+	 * @return mixed if not null, this will be the new value of the plugin hook
+	 */
+	function plugin_hook_handler1(\Elgg\Hook $hook) {
+		...
+	}
+
+	/**
+	 * @param string $hook	The name of the plugin hook
+	 * @param string $type	The type of the plugin hook
+	 * @param mixed  $value	The current value of the plugin hook
```
Comment (jeabakker) [inline docs/design/events.rst]: please check alignment

File: docs/guides/hooks-list.rst [hunk 6]
```diff
@@ -272,10 +272,10 @@ Ajax
...
 	elgg/Ajax method  plugin hook type
 	================  ====================
-	action()          action:<action_name>
-	path()            path:<url_path>
-	view()            view:<view_name>
-	form()            form:<action_name>
+	action()		  action:<action_name>
+	path()			path:<url_path>
+	view()			view:<view_name>
+	form()			form:<action_name>
```
Comment (jeabakker) [inline docs/guides/hooks-list.rst]: how does this align in the file, result is ok

File: docs/guides/notifications.rst [hunk 7]
```diff
@@ -131,41 +131,41 @@ the contents of the notification when a new objects of subtype 'photo' is create
...
 	/**
 	 * Prepare a notification message about a new photo
 	 *
-	 * @param string                          $hook         Hook name
-	 * @param string                          $type         Hook type
+	 * @param string						  $hook		 Hook name
+	 * @param string						  $type		 Hook type
 	 * @param Elgg_Notifications_Notification $notification The notification to prepare
-	 * @param array                           $params       Hook parameters
+	 * @param array							$params		Hook parameters
```
Comment (jeabakker) [inline docs/guides/notifications.rst]: please check alignment

File: docs/guides/notifications.rst [hunk 8]
```diff
@@ -276,11 +276,11 @@ Example:
...
 	 *
-	 * @param string $hook          'get'
-	 * @param string $type          'subscriptions'
+	 * @param string $hook		  'get'
+	 * @param string $type		  'subscriptions'
 	 * @param array  $subscriptions Array containing subscriptions in the form
-	 *                       <user guid> => array('email', 'site', etc.)
-	 * @param array  $params        Hook parameters
+	 *						<user guid> => array('email', 'site', etc.)
+	 * @param array  $params		Hook parameters
```
Comment (jeabakker) [inline docs/guides/notifications.rst]: please check alignment

**Review by jeabakker — COMMENTED** [review]
any idea why github thinks you added 4k line of code. I don't see it

**Pull request closed without merging**